documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,285 @@
+
+**Step #5: Optionally, graph multiple variables updated by the model.**
+
+Using the Flow.js [Variables Channel](../generated/channels/variables-channel/), you can subscribe to multiple model variables at once, which allows you to use multiple variables at once in your visualization.
+
+	Flow.channel.variables.subscribe(['cost', 'price'], 
+		function (data) {
+			var composedData = [
+				{
+					name: 'Cost',
+					data: data.cost
+				},
+				{
+					name: 'Price',
+					data: data.price
+				}
+			];
+			mySecondChart.setData(composedData);
+			mySecondChart.render();
+		}, { batch: true });
+
+Notice that we've included an optional third argument to `subscribe()`: the option `{ batch: true }`. When `batch` is `true`, the callback function is only called once (rather than once for each variable to which you are subscribing). So here, the argument passed to the callback (`data`) is an object with the names and values of both variables. 
+
+In this example, we're assuming that `cost` and `price` are both arrays (for example, tracking cost and price over time as the model advances). Because the updated values are being added as elements to existing array variables, there is no need to explicitly add the updated values onto the data set that we're graphing. (Compare this with Step 4, where for a scalar model variable, we explicitly `push` the new value into the set of data that Contour is graphing.)
+
+**Step #6: Optionally, graph multiple variables entered by the user.**
+
+The approach in Step #5 works well when `cost` and `price` are both updated automatically as your model advances. You can also use Flow.js and Contour to graph multiple model variables whose values are input by the user.
+
+First, have the user enter the values and click a button when finished:
+
+	Enter x-coordinate: <input id="x" type="text"></input><br>
+	Enter y-coordinate: <input id="y" type="text"></input></br>
+	<button id="submitButton">Update x,y</button>
+
+Then once the button is clicked, call an operation from the model, using the values entered by the user:
+
+	$("#submitButton").click(function(){
+		Flow.channel.operations.publish('updateXY', [$("#x").val(), $("#y").val()]);
+	});
+
+We use the `operations.publish()` to guarantee that any subscribers are notified of the call.
+
+Finally, use the Flow.js [Operations Channel](../generated/channels/operations-channel/) to subscribe to the operation you've just called. The second argument is a callback function, called whenever the model operation is called. We want to update and re-render our visualization each time this operation happens:
+
+	Flow.channel.operations.subscribe('updateXY',
+		function(data) {
+			var composedData = {
+				x: data.updateXY.arguments[0],
```

Comment:
To make sure we're talking about the same thing, the flow should be:

inputs -> send to operation -> plot outputs from operation. Inputs -> opn -> plot inputs make the entire operation redundant right? Or am I missing something?

The subscribe call returns whatever the model operation returns. In your case your model (I think) doesn't return anything, which is why you don't have anything else under subscribe. 
I think the fact that this is a contrived example may be part of it - let's call your operation 'forecastProfits' and name the inputs `currentPrice` and  `unitsSold`, and then plot `data.profits` (single series should be fine).

Happy to find 10 mins to do this together if that's easier.